src/Microsoft.DotNet.Wpf/src/PresentationFramework/System/Windows/ResourceDictionary.cs

```diff
@@ -1735,20 +1735,43 @@ private void NotifyOwners(ResourcesChangeInfo info)
                     DeferredResourceReference deferredResourceReference;
                     if (!IsThemeDictionary)
                     {
-                        // Cache the deferredResourceReference so that it can be validated
-                        // in case of a dictionary change prior to its inflation
-                        _deferredResourceReferences ??= new DeferredResourceReferenceList();
-
-                        if (_deferredResourceReferences.Get(resourceKey) is { } existingDeferredResourceReference
-                            && existingDeferredResourceReference.Dictionary == this)
+                        if(FrameworkAppContextSwitches.DisableDynamicResourceOptimization)
```

Comment:
`if(FrameworkAppContextSwitches.DisableDynamicResourceOptimization)`

Can we just make sure all 6 occurrences have space after if?

Reply:
There are incorrect spacing issues in this changeset. I recommend running "format document" in VS
![image](https://github.com/user-attachments/assets/45b2a162-506b-41de-8854-a0732ce8e104)

Reply:
Sure, I will go ahead and do this.
I will try format document, but I am not very keen on it, if it does a lot of formatting.

Reply:
Fixed this